holidayextras/ui-toolkit
# Removing gh-pages from Circle CI

#### What does this PR do? (please provide any background)
- UI-Toolkit has docs which get published to gh-pages branch of GitHub
- CircleCi currently runs tests on that branch, but the branch is a static generated site therefore has no tests.
- This PR stops CircleCI running on that branch.

#### What tests does this PR have?
None, thats kinda the point :)

#### How can this be tested?
- This is going to be a hard one to test, we will need to do (or wait for) a release of the UI Toolkit to see if this runs against the branch.
- It should have no problems going live as it doesn't affect the site, just the CI.

#### Screenshots / Screencast
<img width="209" alt="screen shot 2015-11-18 at 10 20 57" src="https://cloud.githubusercontent.com/assets/778942/11238504/14d26a38-8dde-11e5-9b97-2ede607c2001.png">


#### What gif best describes how you feel about this work?
![Circle](https://media.giphy.com/media/OtP2Jj6IuVbhu/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
